config/application.rb

```diff
@@ -1,6 +1,8 @@
-require_relative "boot"
+# frozen_string_literal: true
 
-require "rails/all"
+require_relative 'boot'
+
+require 'rails/all'
 
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
```

Comment:
- [ ] While running the rails server, a CORS policy error was raised because you didn't allow access to the random messages API you created.

![image](https://github.com/limavfabio/hello-rails-react/assets/68796232/a903ffaf-896f-416a-9cc9-1c4c7c4af698)

Kindly do the following to fix this error:
1. install this gem `rack-cors`.
2. Add this code to the `Application` class here in `config/application.rb`
```ruby
config.middleware.insert_before 0, Rack::Cors do
      allow do
        origins '*'
        resource '*', :headers => :any, :methods => [:get, :post, :options]
      end
    end
```